modules/util/util.js

```diff
@@ -260,7 +260,7 @@ export function utilDisplayLabel(entity, graphOrGeometry, verbose) {
     var presetName = preset && (preset.suggestion ? preset.subtitle() : preset.name());
 
     if (verbose) {
-        result = [presetName, displayName].filter(Boolean).join(' ');
+        result = [`The ${presetName} POI,`, `"${displayName}"`].filter(Boolean).join(' ');
```

Comment:
The `filter` and `join` calls were a clever way to display only `presetName` if `displayName` is empty and vice versa. But now that the array’s only item is always a nonempty string, these calls have no effect. The code needs to handle these edge cases in some other way. While we’re at it, we should make the new format string localizable.